vin/app/controllers/vin/notes_controller.rb

```diff
@@ -1,77 +0,0 @@
-class Vin::NotesController < ApplicationController
-respond_to :json
-
-  # GET /vin/sub/:uid/shipments/:sid/notes
-  def index
-
-    @subscriber = Subscriber.find(params[:sub_id])
-
-    @package = @subscriber.shipments.find(params[:shipment_id]).package
```

Comment:
The Rails way of doing this is with scopes http://guides.rubyonrails.org/active_record_querying.html#scopes

In the `Shipment` model:
```ruby
scope :for_subscriber, -> (subscriber) { where(subscriber: subscriber) }
```
and then here
```ruby
Shipment.for_subscriber(@subscriber).first.package
```